src/WebUI/Pods/PodOverview.tsx

```diff
@@ -49,7 +49,7 @@ export class PodOverview extends BaseComponent<IPodOverviewProps> {
         if (pod.spec && pod.spec.containers && pod.spec.containers.length > 0) {
             const containersCount = pod.spec.containers.length;
             const defaultImage = pod.spec.containers[0].image;
-            image = containersCount > 1 ? localeFormat(Resources.MoreImagesText, defaultImage, containersCount - 1) : defaultImage;
+            image = containersCount > 1 ? Utils.getImageText(pod.spec.containers): defaultImage;
```

Comment:
pass containers to this getImageText() function and let it do all the magic of computation